### ISSUE TITLE
Clean up code and make it less complex/more organized

### ISSUE THREAD

**Issue by snoorkwashanging** [Weeaboo-Bot/discordbot#46]
Code, in general, could use some tidying up. We should go through, simplify, and organize everything more neatly.

**Comment by techie345** [comment]
Agreed. Although I'm planning on working on getting my logs fixed first even in their current state.
